tests/regression

```diff
@@ -95,6 +95,13 @@ idxr_server_start() {
     idxr_server start --web-port "$port" "$@"
 }
 
+idxr_server_start_standard() {
```

Comment:
Function needs to take further arguments that are appended to the command via "$@"

Reply:
Fixed.